.travis.yml

```diff
@@ -5,17 +5,6 @@ scala:
 
 matrix:
   include:
-    - os: linux
-      jdk: oraclejdk8
-      language: scala
-      before_install:
-        - sudo apt-get -qq update
-        - sudo sh -c "echo 'deb http://llvm.org/apt/precise/ llvm-toolchain-precise-3.7 main' >> /etc/apt/sources.list"
-        - sudo sh -c "echo 'deb http://llvm.org/apt/precise/ llvm-toolchain-precise main' >> /etc/apt/sources.list"
-        - wget -O - http://llvm.org/apt/llvm-snapshot.gpg.key | sudo apt-key add -
-        - sudo add-apt-repository --yes ppa:ubuntu-toolchain-r/test
-        - sudo apt-get -qq update
-        - sudo apt-get install -y libgc-dev clang++-3.7 llvm-3.7 llvm-3.7-dev llvm-3.7-runtime llvm-3.7-tool libunwind7-dev
```

Comment:
I don't think we should remove Travis just yet, lets test two CIs alongside in the beginning.